src/components/Movie.jsx

```diff
@@ -0,0 +1,72 @@
+import { useDispatch, useSelector } from 'react-redux'
+import starredSlice from '../data/starredSlice'
+import watchLaterSlice from '../data/watchLaterSlice'
+import placeholder from '../assets/not-found-500X750.jpeg'
+
+const Movie = ({ movie, viewTrailer, closeCard }) => {
+
+    const state = useSelector((state) => state)
+    const { starred, watchLater } = state
+    const { starMovie, unstarMovie } = starredSlice.actions
+    const { addToWatchLater, removeFromWatchLater } = watchLaterSlice.actions
+
+    const dispatch = useDispatch()
+
+    const myClickHandler = (e) => {
+        if (!e) var e = window.event
+        e.cancelBubble = true
+        if (e.stopPropagation) e.stopPropagation()
+        e.target.parentElement.parentElement.classList.remove('opened')
```

Comment:
You should take advantage of React functionality (for example state), and avoid doing DOM manipulation directly with vanilla JS in your components because that could lead to conflicts between the DOM and React virtual DOM.